spring-integration-core/src/main/java/org/springframework/integration/config/xml/GatewayParser.java

```diff
@@ -48,15 +48,16 @@
  */
 public class GatewayParser implements BeanDefinitionParser {
 
+	private static final String NAME_ATTRIBUTE = "name";
```

Comment:
There is a `AbstractBeanDefinitionParser.NAME_ATTRIBUTE` instead which we may consider to reuse...